src/Examples/DAE/petsc_chem_example.ipynb

```diff
@@ -0,0 +1,564 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# First plot all y's on one plot\n",
+    "\n",
+    "a = plt.plot(tj.vecs[\"_time\"], tj.vecs[str(m.y[180, 1])], label=\"y1\")\n",
+    "a = plt.plot(tj.vecs[\"_time\"], tj.vecs[str(m.y[180, 2])], label=\"y2\")\n",
+    "a = plt.plot(tj.vecs[\"_time\"], tj.vecs[str(m.y[180, 3])], label=\"y3\")\n",
+    "a = plt.plot(tj.vecs[\"_time\"], tj.vecs[str(m.y[180, 4])], label=\"y4\")\n",
+    "a = plt.plot(tj.vecs[\"_time\"], tj.vecs[str(m.y[180, 5])], label=\"y5\")\n",
+    "a = plt.plot(tj.vecs[\"_time\"], tj.vecs[str(m.y6[180])], label=\"y6\")\n",
```

Comment:
Rather late on this, but wouldn't it just be easier to use `tj.time` instead of `tj.vecs[\"_time\"]`?